external_services/lti.py

```diff
@@ -19,8 +20,9 @@ class LTIRequest(object):
     def __init__(self, service, user, instance, request, title, context_id=None, link_id=None, add=None, exercise=None):
         self.service = service
         course = instance.course
+        parsed_url = urlparse(settings.BASE_URL)
```

Comment:
Name `parsed_url` doesn't indicate what this variable contains. For example, `parsed_base_url` or `base_url_parts` would indicate information about the content.